<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_window</class>
 <widget class="QMainWindow" name="table_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MenuButton">
    <property name="geometry">
     <rect>
      <x>934</x>
      <y>712</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Главное меню</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>159</y>
      <width>914</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>152</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Организатор</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Место</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="GraphButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewRaceButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>90</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить заезд</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
